include/jwt-cpp/traits/danielaparker-jsoncons/traits.h

```diff
@@ -15,59 +17,103 @@ namespace jwt {
 		/// basic_claim's JSON trait implementation for jsoncons.
 		struct danielaparker_jsoncons {
 			// Needs at least https://github.com/danielaparker/jsoncons/commit/28c56b90ec7337f98a5b8942574590111a5e5831
-			static_assert(jsoncons::version().minor >= 167, "A higher version of jsoncons is required!");
+			static_assert(jsoncons::version().minor >= 167 || (jsoncons::version().major > 0), "A higher version of jsoncons is required!");
```

Comment:
This new approach should work with newer version? Since it no longer uses internal APIs

What version did you test this with?

Reply:
i have tested it with 1.3.0

Reply:
Cool!